README.md

```diff
@@ -7,6 +7,8 @@
 [![https://github.com/psf/black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://black.readthedocs.io/en/stable/)
 [![Documentation Status](https://readthedocs.org/projects/nipoppy/badge/?version=latest)](https://nipoppy.readthedocs.io/en/latest/?badge=latest)
 
+<img alt="Nipoppy logo" src="https://raw.githubusercontent.com/michellewang/nipoppy/224/logo/logo/logo_square.svg" width=100px style="float:right">
```

Comment:
Change image source to `https://raw.githubusercontent.com/nipoppy/nipoppy/main/logo/logo_square.svg`